lib/action_subscriber/dsl.rb

```diff
@@ -1,5 +1,54 @@
+
 module ActionSubscriber
   module DSL
+    class Filter
+      attr_accessor :method
```

Comment:
We should probably use a different name than method since method is a reserved name:
```
irb(main):001:0> class Yolo; end 
=> nil
irb(main):002:0> Yolo.new
=> #<Yolo:0x0000557f0a76da98>
irb(main):003:0> y = Yolo.new
=> #<Yolo:0x0000557f0a761680>
irb(main):004:0> y.method(:to_s)
=> #<Method: Yolo(Kernel)#to_s>
```